### ISSUE TITLE
Buttons have disappeared on Thunderbird 91

### ISSUE THREAD

**Issue by catalizaweb** [thundernest/quicktext#263]
Hi, first of all congratulations for your addon, it is very useful for my work.
I have just updated to Thunderbird 91.0.3 (64-bit) on Windows 10, and your addon to 4.1 version and I can’t see now the quicktext buttons when I compose an email.

![20210901-121121-Window](https://user-images.githubusercontent.com/89909670/131653914-22dc1c28-d5de-4bf6-9c78-b37d52ea97d8.jpg)

Tried to reinstall the addon without success.
I can’t find any option to reactivate them, could you help me please?
Thanks, best regards.

**Comment by SamuelPlentz** [comment]
Have a look here: #261

**Comment by catalizaweb** [comment]
Thanks @SamuelPlentz it is caused by ThunderbirdHTMLedit addon. Disabling that addon fixed it.

**Comment by gresio64** [comment]
After the last update 91.0.3 does the module does not work, how can it be solved?
If uninstalling and reinstalling, I lose all my data from my configurations created?

**Comment by jobisoft** [comment]
Yeah, do not uninstall/reinstall, that will delete your data.

Try to disable other add-ons, maybe they interfere. ThunderbirdHTMLedit seems to be such a candidate.

**Comment by gresio64** [comment]
Thunderbirdhtmledit is not installed
I deactivated the 2 modules, but does not continue to work

**Comment by jobisoft** [comment]
What error do you get?

**Comment by gresio64** [comment]
I don't see errors, I don't see your form in the email

**Comment by gresio64** [comment]
Ah, I forgot, Thunderbird gives me this mistake
QuickText is not compatible with Thunderbird 91.0.3.

**Comment by jobisoft** [comment]
Go to your add-on manager, click on the gear icon and search for updates. Version 4.1 is the latest:
https://addons.thunderbird.net/addon/quicktext/versions/

**Comment by gresio64** [comment]
Ok, it works.
Thanks

**Comment by SamuelPlentz** [comment]
@jobisoft: Added this page to the wiki: [Troubleshooting](https://github.com/jobisoft/quicktext/wiki/Troubleshooting)

Often we have to ask the same questions when the issue is something like: "Doesn't work". So we can reference this wiki page now.

Perhaps we should also show a message when one files a new bug, or perhaps there should be a sticky issue, containing a link to the wiki page.

**Comment by EvilOliverlord** [comment]
> Try to disable other add-ons, maybe they interfere. ThunderbirdHTMLedit seems to be such a candidate.

Following notes in #261, I solved the problem simply by disabling then re-enabling (one by one) all other add-ons. The only one I did not re-enable was duplicateContactsManager, so either that one causes a conflict or the disable/enable cycle solves things.

Note that I had not only lost buttons, but the QuickText template list was always empty the first time I opened it per TB session. Opening it again showed my usual list of templates. Both problems now appear solved.
